src/fitting.jl

```diff
@@ -46,7 +46,8 @@ return the one with minimum error as defined by the algorithm `algo`.
 function fit(::Type{Variogram}, γ::EmpiricalVariogram,
              algo::VariogramFitAlgo=WeightedLeastSquares())
   # fit each variogram type
-  res = [fit_impl(V, γ, algo) for V in STATIONARY]
+  varsubset = filter(x -> x != NuggetEffect, STATIONARY) # excluding NuggetEffect until is properly managed
+  res = [fit_impl(V, γ, algo) for V in varsubset]
```

Comment:
Nice. I think we can do even better here with something like:

```julia
res = [fit_impl(V, γ, algo) for V in STATIONARY if V != NuggetEffect]
```

Can you give it a try?

Reply:
It was my initial implementation but then I made it in that way to facilitate changes later. But it is OK.